api/lib/infrastructure/repositories/competence-repository.js

```diff
@@ -7,44 +7,52 @@ const competenceDatasource = require('../datasources/airtable/competence-datasou
 const knowledgeElementRepository = require('./knowledge-element-repository');
 const scoringService = require('../../domain/services/scoring/scoring-service');
 const { NotFoundError } = require('../../domain/errors');
+const { FRENCH_FRANCE, ENGLISH_SPOKEN } = require('../../domain/constants').LOCALE;
 
 const PixOriginName = 'Pix';
 
-function _toDomain(competenceData, areaDatas) {
+function _toDomain(competenceData, areaDatas, locale) {
   const areaData = competenceData.areaId && _.find(areaDatas, { id: competenceData.areaId });
   return new Competence({
     id: competenceData.id,
-    name: competenceData.name,
+    name: _getTranslatedText(locale, { frenchText: competenceData.nameFrFr, englishText: competenceData.nameEnUs }),
     index: competenceData.index,
-    description: competenceData.description,
+    description: _getTranslatedText(locale, { frenchText: competenceData.descriptionFrFr, englishText: competenceData.descriptionEnUs }),
     origin: competenceData.origin,
     skillIds: competenceData.skillIds,
     area: areaData && new Area({
       id: areaData.id,
       code: areaData.code,
-      title: areaData.titleFrFr,
+      title: _getTranslatedText(locale, { frenchText: areaData.titleFrFr, englishText: areaData.titleEnUs }),
       color: areaData.color,
     }),
   });
 }
 
+function _getTranslatedText(locale, translations = { frenchText: '', englishText: '' }) {
```

Comment:
Qu'est-ce qui fait que l'on ait besoin d'utiliser ça ? Je trouve que ça alourdit la compréhension du code et que RORO serait suffisant 
```suggestion
function _getTranslatedText({ locale, frenchText, englishText }) {
```

Reply:
`{ locale, translations: { frenchText: '', englishText: '' } }`